config/connect-distributed.properties

```diff
@@ -39,4 +39,5 @@ internal.value.converter.schemas.enable=false
 offset.storage.topic=connect-offsets
 # Flush much faster than normal, which is useful for testing/debugging
 offset.flush.interval.ms=10000
-config.storage.topic=connect-configs
\ No newline at end of file
+config.storage.topic=connect-configs
+status.storage.topic=connect-status
```

Comment:
Do we want to file a follow-up JIRA for making all topic settings configurable w/ a simple prefix? No need to roll it into this already large patch, but I don't want to lose track of that idea.